src/main/content/_i18n/en.yml

```diff
@@ -0,0 +1,281 @@
+global:
+  tagline: English
+  openliberty: Open Liberty
+  ol_logo_subtitile: An IBM Open Source Project
+  about: About
+  twitter: Twitter
+  github: GitHub
+  learnmore: Learn more
+  readmore: Read more
+  opentracing: OpenTracing
+  healthcheck: Health Check
+  metrics: Metrics
+  openapi: OpenAPI
+  fault_tolerance: Fault Tolerance
+  jwt: JWT
+  config: Config
+  restclient: Rest Client
+  jaxrs: JAX-RS
+  jsonp: JSON-P
+  jsonb: JSON-B
+  cdi: CDI
+  eclipsemprofile: Eclipse MicroProfile
+  microprofile: MicroProfile
+  microprofile_small_case: microprofile
+  jakartaee: Jakarta EE
+  jakartaee_small_case: jakarta ee
+  zero_migration: zero migration
+  jenkins: Jenkins
+  spring: Spring
+  docker: Docker
+  grafana: Grafana
+  gradle: Gradle
+  logstash: Logstash
+  prometheus: Prometheus
+  maven: Maven
+  kubernetes: Kubernetes
+  kibana: Kibana
+  elastic_search: Elastic Search
+  follow: Follow
+  and: and
+  and_with: And with
+  or: or
+  trans_yes: Yes`
```

Comment:
its not typo it should be within single quotes like 'Yes' because I saw when we just add as Yes yaml doesn't interpret, it throws error
when we add within single quotes output looks like below
<img width="1792" alt="Screenshot 2022-07-20 at 4 55 23 PM" src="https://user-images.githubusercontent.com/87368885/179970856-36b49a72-55c1-4617-9be3-3632637d3168.png">